Kickstarter-iOS/Views/Controllers/SettingsAccountViewController.swift

```diff
@@ -98,6 +102,12 @@ final class SettingsAccountViewController: UIViewController {
       .observeValues { [weak self] _ in
         self?.showChangeCurrencyAlert()
     }
+
+    self.viewModel.outputs.tableViewTopConstraint
```

Comment:
Why was this needed with this change?

Reply:
I don't know if that's the best approach, but this will hide the `emailPassword` section header view. Without updating the constraint, we could see an extra header above the privacy section and it looked weird.

Reply:
Oh I see. I would suggest instead returning a `nil` header view for that section.